pkg/api/handlers/libpod/manifests.go

```diff
@@ -196,13 +195,7 @@ func ManifestInspect(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var schema2List define.ManifestListData
-	if err := json.Unmarshal(rawManifest, &schema2List); err != nil {
-		utils.Error(w, http.StatusInternalServerError, err)
-		return
-	}
-
-	utils.WriteResponse(w, http.StatusOK, schema2List)
+	utils.WriteResponse(w, http.StatusOK, rawManifest)
```

Comment:
The `rawManifest` variable name is now misleading.

Reply:
Renamed.